Wikibase/Sniffs/Commenting/DisallowedDocTagsSniff.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Custom sniff that reports deprecated PHPDoc tags (e.g. known misspellings) and replaces them with
+ * their preferred counterparts.
+ *
+ * @license GPL-2.0+
+ * @author Thiemo Mättig
+ */
+class Wikibase_Sniffs_Commenting_DisallowedDocTagsSniff implements PHP_CodeSniffer_Sniff {
+
+	/**
+	 * @var array Set a value to null if a tag is forbidden, but no automatic fix should be made.
```

Comment:
> The defaults are there, directly below.

Yes, and looking at the first defaults, I have no idea if `@cover` is a misspelling of `@covers` or `@cover` is the correct version of the misspelling `@covers` :) but I guess it’s more clear with the following entries, and allowing the `null` value only makes sense for one of those interpretations.